packages/agents/src/ai-chat-agent.ts

```diff
@@ -245,23 +259,16 @@ export class AIChatAgent<Env = unknown, State = unknown> extends Agent<
   private async _reply(id: string, response: Response) {
     // now take chunks out from dataStreamResponse and send them to the client
     return this._tryCatchChat(async () => {
-      // @ts-expect-error TODO: fix this type error
-      for await (const chunk of response.body!) {
-        const body = decoder.decode(chunk);
-
-        this._broadcastChatMessage({
```

Comment:
why was this broadcast of the text chunks to the client removed? `_drainStream` seems to perform the chunk decode, but never broadcasts the value.